signing_service/signing_service/signing_service.py

```diff
@@ -138,7 +138,11 @@ def run(self) -> None:
                 except (socket.error, socket.timeout) as exception:
                     logger.error(f'Socket error occurred: {exception}')
 
-                    self._attempt_reconnection()
+                    try:
+                        self._attempt_reconnection()
+                    except SigningServiceMaximumReconnectionAttemptsExceeded:
+                        logger.error('Maximum number of reconnection exceeded.')
```

Comment:
`reconnection` -> `reconnections`

Reply:
I have this in my first version, but for some reason Pycharm is marking me this form as a typo. Well. I guess Pycharm is not a vocabulary software after all.
:+1: fixed

Reply:
In singular this sounds very Japanese :)